Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/grpc/grpc-swift.git", from: "1.24.2"),
         .package(url: "https://github.com/stephencelis/SQLite.swift.git", from: "0.15.3"),
-        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", exact: "0.11.0")
+        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", from: "0.12.0")
```

Comment:
This needs to stop being reverted. `from` is _completely incorrect_ to use with Swift for `0.*` dependencies, because Swift's implementation of SemVer is broken for `0.*` versions (it treats them all as compatible because it uses `X.*` as the "breaking change" identifier, but SemVer specifies that anything can change across `0.*` versions).
```suggestion
        .package(url: "https://github.com/Electric-Coin-Company/zcash-light-client-ffi", exact: "0.12.0")
```

Reply:
Exact it is, updated.